pdns/recursordist/pdns_recursor.cc

```diff
@@ -628,12 +629,13 @@ static bool udrCheckUniqueDNSRecord(Logr::log_t nodlogger, const DNSName& dname,
       if (g_udrLog) {
         // This should also probably log to a dedicated file.
         SLOG(g_log << Logger::Notice << "Unique response observed: qname=" << dname << " qtype=" << QType(qtype) << " rrtype=" << QType(record.d_type) << " rrname=" << record.d_name << " rrcontent=" << record.getContent()->getZoneRepresentation() << endl,
-             nodlogger->info(Logr::Debug, "New response observed",
+             nodlogger->info(Logr::Notice, "New response observed",
```

Comment:
intentional change?

Reply:
yes, this was done wrong when SL was added, see the level of the old-style call